spark/Makefile

```diff
@@ -17,10 +17,6 @@ ZEPPELIN_TAG = v0.5.5_v3
 spark:
 	docker build -t gcr.io/google_containers/spark-base base
 	docker tag gcr.io/google_containers/spark-base gcr.io/google_containers/spark-base:$(TAG)
-	docker build -t gcr.io/google_containers/spark-worker worker
-	docker tag gcr.io/google_containers/spark-worker gcr.io/google_containers/spark-worker:$(TAG)
-	docker build -t gcr.io/google_containers/spark-master master
-	docker tag gcr.io/google_containers/spark-master gcr.io/google_containers/spark-master:$(TAG)
 	docker build -t gcr.io/google_containers/spark-driver driver
 	docker tag gcr.io/google_containers/spark-driver gcr.io/google_containers/spark-driver:$(TAG)
```

Comment:
i could go either way on the driver. my decision to leave it was essentially deferring its fate. it would be nice if the driver setup the environment no matter what command is run.